cmds/core/scp/scp.go

```diff
@@ -119,6 +119,7 @@ func scpSink(w io.Writer, r io.Reader, path string) error {
 	return nil
 }
 
+//nolint:unparam
```

Comment:
rather than the no lint: thing, you ought to be able to say
func reply(_ io.Writer, r byte) {...}

If golangci-lint does not understand this, then it does not understand go.